server/src/main/scala/com/cloudera/livy/utils/SparkApp.scala

```diff
@@ -18,15 +18,25 @@
 
 package com.cloudera.livy.utils
 
+import scala.collection.JavaConverters._
+
 import com.cloudera.livy.LivyConf
 import com.cloudera.livy.util.LineBufferedProcess
 
+case class AppInfo(var driverLogUrl: Option[String] = None, var sparkUiUrl: Option[String] = None) {
+  def asJavaMap: java.util.Map[String, String]
+    = Map("driverLogUrl" -> driverLogUrl.orNull, "sparkUiUrl" -> sparkUiUrl.orNull).asJava
```

Comment:
Can we define these keys "driverLogUrl", "sparkUiUrl" in a common place. I found that lots of places use them, and it easily introduces the typo.